.drone.yml

```diff
@@ -66,6 +66,16 @@ steps:
   when:
     event: tag
 
+- name: publish:gcs
+  image: plugins/gcs
+  settings:
+    source: gcs_bucket
+    target: codesrv-ci.cdr.sh/
```

Comment:
not sure if this is the correct bucket for this, I think this is the same we used for nbin AFAIK?

Reply:
This is the correct bucket